charts/ingress-nginx/templates/controller-ingressclass.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.controller.ingressClassResource.enabled -}}
+apiVersion: networking.k8s.io/v1
+kind: IngressClass
+metadata:
+  labels:
+    {{- include "ingress-nginx.labels" . | nindent 4 }}
+    app.kubernetes.io/component: controller
+    {{- with .Values.controller.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  name: {{ .Values.controller.ingressClass }}
+{{- if .Values.controller.ingressClassResource.default -}}
+  annotations:
+    ingressclass.kubernetes.io/is-default-class: "true"
+{{- end }}
+spec:
+  controller: k8s.io/ingress-nginx
+  parameters: {}
```

Comment:
I believe parameters can be removed if it will not contain a value

Reply:
makes sense